templates/home.html

```diff
@@ -1,16 +0,0 @@
-{% extends 'layout.html' %}
```

Comment:
Is there a reason why we want to remove the home page?

Reply:
Hmm, I think I was getting ahead of myself, perhaps:

In the [Figma for the visual update](https://www.figma.com/file/PchgLiEwThKX48XA04tPLw/movr-web-application_v1?node-id=235%3A0), the home page no longer exists:

<img width="300" src="https://user-images.githubusercontent.com/3051672/87685070-f7fbb200-c750-11ea-8932-e4f5f389a714.png">

But your note about having location information is a good point. So why don't I modify this PR to leave the home page as-is. That's also better from a PR scoping standpoint, as then this PR will be smaller/cleaner (i.e. _just_ be about setting the active nav header).